Author Luca Boccassi <luca.boccassi@gmail.com>
Description: test: customize /etc/os-release instead of /usr/lib/os-release
Origin: upstream, commit:2f6fe4e1131d39fcafa9e00a7902919efb5361e1
--- a/test/units/TEST-82-SOFTREBOOT.sh
+++ b/test/units/TEST-82-SOFTREBOOT.sh
@@ -157,9 +157,9 @@ elif [ -f /run/TEST-82-SOFTREBOOT.touch ]; then

     # Copy os-release away, so that we can manipulate it and check that it is updated in the propagate
     # directory across soft reboots. Try to cover corner cases by truncating it.
-    mkdir -p /tmp/nextroot-lower/usr/lib
-    grep ID /etc/os-release >/tmp/nextroot-lower/usr/lib/os-release
-    echo MARKER=1 >>/tmp/nextroot-lower/usr/lib/os-release
+    mkdir -p /tmp/nextroot-lower/etc
+    grep ID /etc/os-release >/tmp/nextroot-lower/etc/os-release
+    echo MARKER=1 >>/tmp/nextroot-lower/etc/os-release
     cmp /etc/os-release /run/systemd/propagate/.os-release-stage/os-release
     (! grep -q MARKER=1 /etc/os-release)

